tests/FunctionalTest/ContainerInteractionTest.php

```diff
@@ -25,7 +25,7 @@ class ContainerInteractionTest extends AbstractFunctionalTest
     /**
      * @test Get a Symfony entry from PHP-DI's container
      */
-    public function phpdi_should_get_entries_from_symfony()
+    public function phpdiShouldGetEntriesFromSymfony()
```

Comment:
Could you restore the original code? (including in the other tests)

Reply:
Done